pyProbeParticle/ocl/AFMulator.py

```diff
@@ -422,7 +430,7 @@ def quick_afm(file_path, scan_size=(16, 16), offset=(0, 0), distance=8.0, scan_s
         sigma: float. Width of tip charge distribution.
         num_heights: int. Number of different heights to scan.
         amplitude: float. Oscillation amplitude in angstroms.
-        out_dir: str or None. Output folder path. If None, defaults to afm_ + input_file_name.
+        out_dir: str or None. Output folder path. If None, defaults to "./afm\_" + input_file_name.
```

Comment:
Why is there the backslach after "afm"?

Otherwise it is completely OK with me.

Reply:
The backslash is an escape character here. Without it, the 'afm' string is interpreted as a hyperlink on the html page.

Reply:
I see, good,